### DISCUSSION TITLE
Is there a way to export renovate output to a json file or something similar?

### DISCUSSION THREAD

**Discussion by cavadalizada** [renovatebot/renovate#21991] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Create a .JSON file, with file path, current version, updated version, label.

### What you tried so far.

I have tried research default renovate options, haven't had luck.

**Comment by rarkins** [comment, marked as answer]
https://docs.renovatebot.com/self-hosted-configuration/#logfile
